client/my-sites/customer-home/cards/features/blogging-prompt/prompts-navigation.jsx

```diff
@@ -48,17 +52,20 @@ const PromptsNavigation = ( { prompts } ) => {
 		}
 	};
 
-	const trackBloggingPromptClick = () => {
+	const handleBloggingPromptClick = ( e ) => {
+		// Prevent navigating away so we have time to record the click.
+		e.preventDefault();
+
 		dispatch(
 			recordTracksEvent( `calypso_customer_home_answer_prompt`, {
 				site_id: siteId,
-				prompt_id: getPrompt().id,
+				prompt_id: getPrompt()?.id,
 			} )
 		);
 
 		// Track if a user skipped todays prompt and choose to answer another prompt
 		const todayPromptId = prompts[ 0 ].id;
-		const skippedPromptId = getPrompt().id;
+		const skippedPromptId = getPrompt()?.id;
 		if ( todayPromptId !== skippedPromptId ) {
```

Comment:
what's recorded as `prompt_id` is currently the id of the prompt that they chose. I think we want to be logging the id of the prompt that they skipped, which would be `todayPromptId`.

The variable `skippedPromptId` is confusingly named, it's actually the _selected_ prompted id.

Reply:
Great catch, thank you! Updated.

Reply:
Good catch alright. I was trying to log what prompt the user skipped to but I see now that we already log that in the `calypso_customer_home_answer_prompt` event.